var/spack/repos/builtin/packages/e4s-cl/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class E4sCl(PythonPackage):
+    """Container Launcher for E4S containers, facilitating MPI library
+    translations"""
+
+    maintainers = ["spoutn1k", "FrederickDeny"]
+    homepage = "https://e4s-cl.readthedocs.io"
+    url = "https://oaciss.uoregon.edu/e4s/e4s-cl/releases"
+    git = "https://github.com/E4S-Project/e4s-cl"
```

Comment:
Not sure how others feel about this but I usually prefer checksummable URLs over git commits.

Reply:
I originally had it that way but I figured git commits could prove more reliable than archives manually copied to another server. I can change it if necessary.